client/state/data-layer/wpcom/read/lists/delete/index.js

```diff
@@ -26,8 +27,15 @@ registerHandlers( 'state/data-layer/wpcom/read/lists/delete/index.js', {
 					},
 					action
 				),
-			onSuccess: noop,
-			onError: () => errorNotice( translate( 'Unable to remove list' ) ),
+			onSuccess: () => {
+				return [
+					navigate( `/read` ),
+					successNotice( translate( 'List deleted successfully.' ), {
+						duration: DEFAULT_NOTICE_DURATION,
```

Comment:
If it's the default, you shouldn't have to specify it :) 
But maybe a future PR to default everything to 5s.

Reply:
True :) Would `STANDARD_NOTICE_DURATION` be better? I notice (ha!) we usually choose `5000` but the value varies and is sprinkled throughout the data layer. Or an argument for `autoDismiss` on `successNotice` that sets a standard duration?

I think in some circumstances we might not want the notice to automagically disappear - for an important notice that really needs to be acknowledged perhaps.